src/main/java/am/ik/yavi/constraint/temporal/LocalTimeConstraint.java

```diff
@@ -0,0 +1,89 @@
+package am.ik.yavi.constraint.temporal;
+
+import am.ik.yavi.core.ConstraintPredicate;
+
+import java.time.LocalTime;
+import java.util.function.Predicate;
+
+import static am.ik.yavi.core.NullAs.VALID;
+import static am.ik.yavi.core.ViolationMessage.Default.TEMPORAL_HOUR;
+import static am.ik.yavi.core.ViolationMessage.Default.TEMPORAL_MINUTE;
+
+/**
+ * This is the actual class for constraints on LocalTime.
+ *
+ * @author Diego Krupitza
+ */
+public class LocalTimeConstraint<T>
+		extends ComparableTemporalConstraintBase<T, LocalTime, LocalTimeConstraint<T>> {
+
+	/**
+	 * Is the given LocalTime at the same hour as {@code other}
+	 *
+	 * @param other the other localtime that wraps the hour
+	 * @since 0.10.0
+	 */
+	public LocalTimeConstraint<T> hour(LocalTime other) {
+		this.predicates().add(ConstraintPredicate.of(x -> x.getHour() == other.getHour(),
+				TEMPORAL_HOUR, () -> new Object[] { other.getHour() }, VALID));
+		return cast();
+	}
+
+	/**
+	 * Is the given LocalTime at the same hour as {@code hour}
+	 *
+	 * @param hour the hour the LocalTime should have
+	 * @since 0.10.0
+	 */
+	public LocalTimeConstraint<T> hour(Integer hour) {
```

Comment:
I don't think constraints on fixed values are that useful.
Instead, I want constraints such as whether the time is in 15-minute increments (0 or 15 or 30 or 45).

Reply:
More generically, it might be nice (not necessary) to also have methods that take `Predicate<Integer>` to check if a given rule of minutes or time is met.

Reply:
I think constraints on fixed values can be quite useful. For example I recently worked on an application where (due to usability reasons) deadlines had to happen always 1 min before the full hour (aka XX:59) therefore we had to validate a fixed minute.

Reply:
I can also think about another use case. Let's say registration time for uni courses. Some faculties only allow the registration to be from 8:00 till 8:59. We still could use a range but just saying "hour(8)" would be easier and cleaner to write.

Reply:
Thanks for the feedback.
Let’s keep it.

Reply:
`Predicate<Integer>` for hour and minute is not possible since I added `Predicate<LocalTime>`. This makes more sense to me because of `LocalTime#now()`.

Reply:
FYI: You added not `Predicate<LocalTime>` but `Supplier<LocalTime>`.